lib/streamlit/connections/sql_connection.py

```diff
@@ -135,7 +142,10 @@ def query(
             ),
             wait=wait_fixed(1),
         )
-        @cache_data(ttl=ttl)
+        @cache_data(
+            show_spinner="Running `SQL.query(...)`.",
```

Comment:
nit: maybe `sql.query(...)` would be slightly better here since it's more likely to be something called on an instance, and the lowercase version also corresponds better to the connection name.